log.json

```diff
@@ -0,0 +1 @@
+{"16-08-2024": []}
```

Comment:
saw this... what is this for?

Reply:
This is an example of the log.json file that is stored on the S3 bucket that keeps track of what files in the day have already been downloaded. It's empty in this case as no cases have been added for this date. This file is stored on the S3 when running on the cloud so is not needed as part of the pipeline, will remove thank you.